test/resolveGitDir.spec.js

```diff
@@ -13,14 +12,4 @@ describe('resolveGitDir', () => {
     expect(resolveGitDir('.')).toEqual(expected)
```

Comment:
`resolveGitDir` no longer accepts an argument. Please remove the parameter `'.'` in the function invocation.